### ISSUE TITLE
The game cannot be started. 

### ISSUE THREAD

**Issue by vangod90** [mknurs/kriegsspiel#1]
Can you give a short introduction how to start the game?

**Comment by mknurs** [comment]
Hello,
sorry I missed your issue for so long.

The game is currently lacking in accessibility and clarity of rules and procedures, but it can be played by two humans (there is no AI yet!).

In the lobby screen you set your username and roomname. An uuid is generated for you, but you can set the roomname to any unique string you'd like. Your enemy should enter the same room on their side. When two players are present in the same room, the "placement" mode begins. When you're ready click ready. When both players are ready battle begins. This applications of Debord's Game of war is still in it's early development.

Currently, i'd suggest reading the rules here, in this pdf: http://www.classwargames.net/wp-content/uploads/2016/01/GoW-Script-for-web.pdf
